frontend/src/store/lessons.ts

```diff
@@ -1,52 +1,50 @@
 import { createSlice } from '@reduxjs/toolkit';
 
-interface LessonResponse {
-  id:number;
-  word:string;
-  category_id:number;
-  created_at: Date;
-  updated_at:Date;
+export interface Lesson {
+  id: number;
+  word: string;
+  category_id: number;
+  choices: Choice[]
 }
 
+export interface Choice {
+  id: number;
+  name: string;
+  lesson_id: number;
+  is_correct: boolean;
+}
+
+
 interface LessonState {
-  lessons: LessonResponse[],
+  lessons: Lesson[],
   isLoading: boolean,
-  isError: boolean,
-  isInvalid: boolean,
   isAddingLesson: boolean
 }
 
 const initialState: LessonState = {
   lessons: [],
   isLoading: false,
-  isError: false,
-  isInvalid: false,
   isAddingLesson: false
 };
 
 export const lessonSlice = createSlice({
   name: 'lesson',
   initialState: initialState,
   reducers: {
-    getAll: (state, action) => {
-      state.lessons = action.payload;
+    getLessons: (state, action) => {
+      state.lessons = action.payload;      
```

Comment:
Remove trailing space
```suggestion
      state.lessons = action.payload;
```